web/views/metrics.erb

```diff
@@ -2,7 +2,7 @@
 <script type="text/javascript" src="<%= root_path %>javascripts/chartjs-plugin-annotation.min.js"></script>
 <script type="text/javascript" src="<%= root_path %>javascripts/metrics.js"></script>
 
-<h2>Total execution time</h2>
+<h2>Metrics</h2>
```

Comment:
Note that I changed the header to something generic, since the actual metric can change. I didn't think it was worth the extra complexity to wire this up via JS.